DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs

```diff
@@ -241,20 +252,53 @@ public void Get_tutorial_information_nextSection_skips_sections_full_of_archived
         [Test]
         public void Get_tutorial_information_nextSection_returns_section_with_only_diagnostic_assessment()

```

Comment:
Why did this test need to change? 🙂

Reply:
This test wasn't precise enough, the next section being tested here would also be included because it has a post-learning assessment, not just because of the diagnostic assessment, until I temporarily removed that assessment with this change 🙂

Reply:
I see, thanks for explaining 🙂